bot/discordPages.go

```diff
@@ -114,6 +109,7 @@ func ReactionListen(session *discordgo.Session, reaction *discordgo.MessageReact
 			pageListeners[reaction.MessageID].CurrentPage++
 			session.ChannelMessageEditEmbed(reaction.ChannelID, reaction.MessageID, &discordgo.MessageEmbed{
 				Title:       pageListeners[reaction.MessageID].Type,
+				URL:         pageListeners[reaction.MessageID].Data.URL,
```

Comment:
I do like this PR but I'm confused about your change here, is there a reason why you added the URL field here and not in the original sending of the embed and/or to the `leftArrow` case?

Reply:
I think I just forgot, I'll look at it tomorrow.
Being able to go to the docs directly for further docs or example would be appreciated, so I thought I added it but it looks like I forgot a few embeds